src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -70,69 +70,61 @@ The `NormOneCone` is representable with LP constraints, since
 ``t \\ge \\sum_i \\lvert x_i \\rvert`` if and only if there exists a vector y such that
 ``t \\ge \\sum_i y_i`` and ``y_i \\ge x_i``, ``y_i \\ge -x_i`` for all ``i``.
 """
-struct NormOneBridge{T, F, G, H} <: AbstractBridge
+struct NormOneBridge{T, F, G} <: AbstractBridge
     y::Vector{MOI.VariableIndex}
-    ge_index::CI{F, MOI.GreaterThan{T}}
-    nn_index::CI{G, MOI.Nonnegatives}
+    nn_index::CI{F, MOI.Nonnegatives}
 end
-function bridge_constraint(::Type{NormOneBridge{T, F, G, H}}, model::MOI.ModelLike, f::H, s::MOI.NormOneCone) where {T, F, G, H}
+function bridge_constraint(::Type{NormOneBridge{T, F, G}}, model::MOI.ModelLike, f::G, s::MOI.NormOneCone) where {T, F, G}
     f_scalars = MOIU.eachscalar(f)
     d = MOI.dimension(s)
     y = MOI.add_variables(model, d - 1)
-    ge_index = MOIU.normalize_and_add_constraint(model, MOIU.operate(-, T, f_scalars[1], MOIU.operate(sum, T, y)), MOI.GreaterThan(zero(T)), allow_modify_function=true)
+    ge = MOIU.operate(-, T, f_scalars[1], MOIU.operate(sum, T, y))
     lb = f_scalars[2:d]
     ub = MOIU.operate(-, T, lb)
     lb = MOIU.operate!(+, T, lb, MOI.VectorOfVariables(y))
     ub = MOIU.operate!(+, T, ub, MOI.VectorOfVariables(y))
-    f_new = MOIU.operate(vcat, T, ub, lb)
-    nn_index = MOI.add_constraint(model, f_new, MOI.Nonnegatives(2d - 2))
-    return NormOneBridge{T, F, G, H}(y, ge_index, nn_index)
+    f_new = MOIU.operate(vcat, T, ge, ub, lb)
+    nn_index = MOI.add_constraint(model, f_new, MOI.Nonnegatives(2d - 1))
+    return NormOneBridge{T, F, G}(y, nn_index)
 end
 
-MOI.supports_constraint(::Type{NormOneBridge{T}}, ::Type{<:MOI.AbstractVectorFunction}, ::Type{MOI.NormOneCone}) where T = true
+MOI.supports_constraint(::Type{<:NormOneBridge{T}}, ::Type{<:MOI.AbstractVectorFunction}, ::Type{MOI.NormOneCone}) where T = true
```

Comment:
We don't use `<:` apart from the indicator bridge (which should be changed). I'd rather keep the same thing for everywhere to ensure a consistent behavior of bridges